README.md

```diff
@@ -1,2 +1,3 @@
+[![Build Status](https://travis-ci.com/kpi-web-guild/django-girls-blog-serhii73.svg?branch=master)](https://travis-ci.com/kpi-web-guild/django-girls-blog-serhii73)
 # django-girls-blog-serhii73
-##### This is a simple blog based on the djangogirl's tutorial.
\ No newline at end of file
+##### This is a simple blog based on the djangogirl's tutorial.
```

Comment:
Why did you change this?

Reply:
I solved the conflict in the browser and probably I pressed the key.
In the browser, it's not very convenient to do. My fault.

Reply:
You can as well do this using Git CLI.